apps/survey/src/components/elements/TimePicker.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
+    <v-combobox
```

Comment:
I think combox also allows to enter values manually / typed-in, if it's intention, you willl need to validate output so the typed-in time format is correct. If only select is intended, simple `<v-select>` can be used